numpy/core/setup.py

```diff
@@ -381,9 +381,9 @@ def check_mathlib(config_cmd):
             mathlibs = libs
             break
     else:
-        raise EnvironmentError("math library missing; rerun "
-                               "setup.py after setting the "
-                               "MATHLIB env variable")
+        raise RuntimeError(
+            "math library missing; rerun setup.py after setting the "
+            "MATHLIB env variable")
```

Comment:
This exception was change to RuntimeError, as it doesn't seem to be an OS-related issue.

Reply:
Maybe `FileNotFoundError`? That doesn't seem quite right either but it is merged with `OSError`.